app/controllers/home_controller.rb

```diff
@@ -38,6 +38,9 @@ def about
       image_tag = ActionController::Base.helpers.image_tag(image_path)
       [img_file, image_tag.tr('"', "'")]
     end.to_h
+    @about_users = AboutUser.all.order("RANDOM()")
```

Comment:
If this variable is not passed down to the view, you can just use `about_users`

Reply:
👍